pom.xml

```diff
@@ -36,8 +36,8 @@
         </plugin> 
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.5</version>
+        <artifactId>maven-plugin-plugin</artifactId>
```

Comment:
? `maven-plugin-plugin` is for writing maven plugins no?  doesn't have equiv functionality as the `maven-release-plugin`.  What is the error you're getting?